pulsar-client/src/main/java/org/apache/pulsar/client/api/Message.java

```diff
@@ -124,4 +124,14 @@
      * @return the key of the message
      */
     String getKey();
+
+    /**
+     * Get the topic name of this message.
+     * This is mainly for TopicsConsumerImpl to identify a message belongs to which topic.
```

Comment:
"TopicsConsumerImpl" we shouldn't be referring to implemenation classes from the API javadoc

Reply:
Thanks. will change it